management/ia/rules_test.go

```diff
@@ -359,7 +359,7 @@ func TestRulesAPI(t *testing.T) {
 		}
 	})
 
-	t.Run("list", func(t *testing.T) {
+	t.Run("list without pagination", func(t *testing.T) {
```

Comment:
Does that mean that without pagination we will return all alertRules?

Reply:
no, just the first page with default page size 
https://github.com/percona/pmm-managed/pull/642/files#diff-3579dfaf2503ed8fb00f2bfe06ee104c8c5bbb9d89c3a59fbd7afa52ecf3ed46R259

Reply:
resolving